src/Propel/Runtime/ActiveQuery/ModelCriteria.php

```diff
@@ -2088,20 +2089,17 @@ protected function getColumnFromName($phpName, $failSilently = true)
             throw new UnknownModelException(sprintf('Unknown model, alias or table "%s"', $prefix));
         }
 
-        if ($tableMap->hasColumnByPhpName($phpName)) {
-            $column = $tableMap->getColumnByPhpName($phpName);
+        $column = ($tableMap->hasColumnByPhpName($phpName)) ? $tableMap->getColumnByPhpName($phpName)
```

Comment:
Could you extract that in a more readable method?

Reply:
Added a method `findColumnByName()` on TableMap. I also changed the variable name from `$phpName` to `$columnName`, apparently it does not have to be a phpName.